### PR TITLE
Adding more info

### PR DESCRIPTION
**Pull request by get-itips**
@dariomws What do you think? Thanks
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2761

### CODE REVIEW

File: Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/manage-policy-tips.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Policy Tips are informative notices that are displayed to email senders while th
 
 - Policy Tips will only show up for email senders when the following conditions are met:
 
-1. Sender's message client program is Microsoft Outlook 2013. If your organization has deployed Exchange 2013 SP1 or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
+1. Sender's message client program is Microsoft Outlook 2013 or later, ([this article](https://support.office.com/article/outlook-license-requirements-for-exchange-features-46b6b7c5-c3ca-43e5-8424-1e2807917c99?ui=en-US&rs=en-001&ad=US) lists the Outlook client versions/licenses that support DLP policy tips). If your organization has deployed Exchange 2013 SP1 or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
```
Comment (dariomws) [inline Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/manage-policy-tips.md:35]:
```suggestion
1. Sender's message client program is Microsoft Outlook 2013 or later, ([this article](https://support.office.com/article/outlook-license-requirements-for-exchange-features-46b6b7c5-c3ca-43e5-8424-1e2807917c99) lists the Outlook client versions/licenses that support DLP policy tips). If your organization has deployed Exchange 2013 SP1 or later, or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
```

**Review by dariomws — COMMENTED** [review]
Looks good but added some suggestions.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/manage-policy-tips.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Policy Tips are informative notices that are displayed to email senders while th
 
 - Policy Tips will only show up for email senders when the following conditions are met:
 
-1. Sender's message client program is Microsoft Outlook 2013. If your organization has deployed Exchange 2013 SP1 or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
+1. Sender's message client program is Microsoft Outlook 2013 or later, ([this article](https://support.office.com/article/outlook-license-requirements-for-exchange-features-46b6b7c5-c3ca-43e5-8424-1e2807917c99) lists the Outlook client versions/licenses that support DLP policy tips). If your organization has deployed Exchange 2013 SP1 or later, or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/manage-policy-tips.md:35]:
```suggestion
1. Sender's message client program is Microsoft Outlook 2013 or later. (For a list of the Outlook client versions/licenses that support DLP policy tips, see ([Outlook license requirements for Exchange features](https://support.office.com/article/outlook-license-requirements-for-exchange-features-46b6b7c5-c3ca-43e5-8424-1e2807917c99).) If your organization has deployed Exchange 2013 SP1 or later, or is using Exchange Online, Policy Tips also show up in Outlook on the web (formerly known as Outlook Web App) and OWA for Devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**